source/Ship.cpp

```diff
@@ -2814,6 +2814,43 @@ double Ship::Energy() const
 
 
 
+// Determine if the player has a jump drive fuel cost display installed
+bool Ship::DisplayMass() const
+{
+	bool displayMass = attributes.Get("Mass Display");
```

Comment:
So normal conditions are lowercase. Might be something we want to keep consistent?
This applies to the other new attributes as well.

Reply:
Yes, they should be.

Reply:
Done. Thanks for reminding me!